ddtrace/internal/datadog/profiling/stack_v2/__init__.py

```diff
@@ -0,0 +1,33 @@
+try:
+    from ._stack_v2 import *  # noqa: F401, F403
+
+    def is_available():
```

Comment:
If we make this a module attribute instead we could probably save a function call